receipt_upload/receipt_upload/merchant_resolution/resolver.py

```
@@ -145,6 +273,237 @@ def resolve(
 
         return result
 
+    def _get_line_for_phone(
+        self,
+        words: List[ReceiptWord],
+        lines: List[ReceiptLine],
+        phone: str,
```

Ruff (0.14.11)

Unused method argument: `phone`

(ARG002)